app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -208,6 +237,13 @@ class ChannelPostList extends PureComponent {
                 >
                     <ChannelLoader theme={theme}/>
                 </AnimatedView>
+                <AnimatedView style={{position: 'absolute', top: 0, height: retryMessageHeight, width: deviceWidth, backgroundColor: '#fb8000', flexDirection: 'row', alignItems: 'center', paddingHorizontal: 10}}>
```

Comment:
Can we put any of this in a stylesheet and pass in multiple styles like `style={[style.retry, {height: retryMessageHeight}]}`